content/en/infrastructure/faq/set-up-orchestrator-explorer-daemonset.md

```diff
@@ -112,4 +151,4 @@ In some setups, the Process Agent and Cluster Agent cannot automatically detect
   ```
 
   [1]: /containers/cluster_agent/
-  [2]: /containers/cluster_agent/setup/?tab=daemonset#pagetitle
\ No newline at end of file
+  [2]: /containers/cluster_agent/setup/?tab=daemonset#pagetitle
```

Comment:
```suggestion
  [2]: /containers/cluster_agent/setup/?tab=daemonset
```

Reply:
Not sure why it wants to add #pagetitle to the link, but it looks like it still works without it so suggesting to remove it!

Reply:
It's weird, I haven't made any changes on it. Anyway, I'm all good to remove it!